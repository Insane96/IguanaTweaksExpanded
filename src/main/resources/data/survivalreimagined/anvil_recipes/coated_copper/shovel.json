{
	"item_to_repair": {
		"id": "survivalreimagined:coated_copper_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 1,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 2
		}
	]
}