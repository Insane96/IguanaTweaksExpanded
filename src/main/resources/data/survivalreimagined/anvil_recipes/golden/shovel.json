{
	"item_to_repair": {
		"id": "minecraft:golden_shovel"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 1
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 9
		}
	]
}